ane_transformers/huggingface/test_distilbert.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (C) 2022 Apple Inc. All Rights Reserved.
 #
 
+import einops
```

Comment:
this looks like a 3rd party dependency that needs adding to requirements.txt + setup.py ?